docs/tests.md

```diff
@@ -12,7 +12,12 @@ found in the [general test directory](/test/general).
 
 In order to run tests from your world, you will need to create a `test` package within your world package. This can be
 done by creating a `test` directory with a file named `__init__.py` inside it inside your world. By convention, a base
```

Comment:
Should we specify that this `__init__.py` is preferably an empty file?

Reply:
I am assuming you want to reduce confusion for people that don't know what an (empty) `__init__.py` is doing?

I think it's generally fine to put *something* in it, so that wording may not be ideal, i.e.
* I could imagine that Stardew's run_long_tests could be set in there.
* Some people put a single comment into all of their `__init__.py`s